kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -65,14 +125,26 @@ To run the generated native executable, generated in `target/`, execute
 
 ### Use the application
 
-To make use of this application it is as simple as putting a message on `travellers` topic with following content (cloud event format) 
+To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* To examine ProcessedTravellers topic and verify upcoming messages will be processed
+
+```
+kafka_2.11-2.4.0/bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers 
+```
+
+* Send Message to Topic
+```
+kafka_2.11-2.4.0/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
```

Comment:
You can remove the `kafka_2.11-2.4.0/` part

Reply:
fixed @radtriste